src/common/serviceInstanceHelpers.js

```diff
@@ -31,6 +31,18 @@ class AMQServiceInstanceTransform {
   }
 }
 
+class CRUDAppInstanceTransform {
```

Comment:
Can we pull these classes out into there own files and keep them in sub directory (e.g. src/common/services/)?

Reply:
Will do this in future PR.